test/ui/ui-test-users-id-tasks-id.js

```diff
@@ -0,0 +1,16 @@
+const puppeteer = require("puppeteer");
+
+(async () => {
+  const browser = await puppeteer.launch(); //promise to launch puppeteer
+  const page = await browser.newPage(); //promise to open a browser
+  await page.goto("https://nscstrdevusw2thucommon.z5.web.core.windows.net/users/1/tasks/1"); //promise to go to the specified page
+  
+
+  let universaldate = new Date(); //   capture current local date and time in universal time
+  let stringDate = universaldate.getFullYear() + "-" + (universaldate.getMonth() + 1) + "-" + universaldate.getDate() + "-" + universaldate.getHours() + ":" + universaldate.getMinutes() + ':' + universaldate.getSeconds();
```

Comment:
@natni and I are using yyyy_mm_dd_hh:mm:ss, so it might be good to switch for consistency, though this is definitely a nitpick and not really an essential change by any means.

Reply:
updated